test/types/index.tst.ts

```diff
@@ -1,87 +1,121 @@
-import { expect, test } from 'tstyche'
+import { describe, expect, test } from 'tstyche'
 
-import { InternalTypedDocument, Orama, PartialSchemaDeep, Results, Schema, SearchParams, TypedDocument } from '@orama/orama'
+import { type InternalTypedDocument, type Orama, type PartialSchemaDeep, type Results, type Schema, type SearchParams, type TypedDocument } from '@orama/orama'
```

Comment:
```suggestion
import type { InternalTypedDocument, Orama, PartialSchemaDeep, Results, Schema, SearchParams, TypedDocument } from '@orama/orama'
```

Reply:
mmh, `type` is better than no `type` because we are just using them as type in the file. IMHO, it's more correct with type. But if you prefer, without letting me know, I can remove it.

Reply:
done!